src/hammer-tech/hammer_tech.py

```diff
@@ -386,7 +387,12 @@ def load_from_json(cls, technology_name: str, json_str: str, path: str) -> "Hamm
         :param path: Path to set as the technology folder (e.g. foo/bar/technology/asap7)
         """
 
-        tech = HammerTechnology()
+        # try to override tech, if __init__.py exist.
```

Comment:
If we are to make this change we should document somewhere this format for `__init__.py`:

```python
from hammer_tech import HammerTechnology

class MyTech(HammerTechnology):
  ...

tech = MyTech()
```

Reply:
What's your suggestion? I think currently we can leave `technology/asap/__init__.py` as an example is ok?

Reply:
Yeah leaving the asap7 example is fine; we may also want to add it to a readme - maybe `src/hammer-tech/README.md`?

Reply:
I just tested the `sphinx` doc generation, it works well, I think we may not need to add a `src/hammer-tech/README.md`

Reply:
Hmm, how about this?

1) Let's put this sketch into the docstring for the `HammerTechnology` class

2) Can we add a unit test that tests this behaviour?

Reply:
great! working on this now.